src/app/clusters/scenes/SceneTableImpl.cpp

```diff
@@ -206,13 +261,17 @@ struct FabricSceneData : public PersistentData<kPersistentFabricBufferMax>
 
         ReturnErrorOnFailure(reader.Next(TLV::ContextTag(TagScene::kSceneCount)));
         ReturnErrorOnFailure(reader.Get(scene_count));
+
+        // In the event of an OTA, is max_scenes_per_fabric was reduced, this will return an error signaling the load function that
```

Comment:
```suggestion
        // In the event of an OTA, if max_scenes_per_fabric was reduced, this will return an error signaling the load function that
```